src/site/content/en/fast/lazy-loading-video/index.md

```diff
@@ -35,15 +35,18 @@ on the `<video>` element may be desirable:
 </video>
 ```
 
+{% Aside 'important' %}
+Video `poster` images may be considered [LCP candidates](/lcp/#what-elements-are-considered). If you decide to lazy load a video's `poster` image, make sure you're not lazy loading them with a JavaScript-based solution (such as [yall.js mentioned below](#libraries)) if they're in the viewport during startup. This is because the [browser preload scanner](/preload-scanner/#lazy-loading-with-javascript) can't discover them as soon as possible, as they will rely on [a data attribute](https://developer.mozilla.org/docs/Learn/HTML/Howto/Use_data_attributes).
+{% endAside %}
+
 The example above uses a `preload` attribute with a value of `none` to prevent browsers
 from preloading _any_ video data. The `poster`
 attribute gives the `<video>` element a placeholder that will occupy the space while the video loads. The reason for this is
 that default behaviors for loading video can vary from browser to browser:
 
 - In Chrome, the default for `preload` used to be `auto`, but as of Chrome 64, it now
 defaults to `metadata`. Even so, on the desktop version of Chrome, a portion of
-the video may be preloaded using the `Content-Range` header. Firefox, Edge and
-Internet Explorer 11 behave similarly.
+the video may be preloaded using the `Content-Range` header. Other Chromium-derived browsers and Edge behave similarly.
```

Comment:
Think we we normally "chromium-based". Also you said Edge when I think you meant Firefox.

```suggestion
the video may be preloaded using the `Content-Range` header. Other Chromium-based browsers and Firefox behave similarly.
```

Reply:
You resolved this but the Edge/Firefox comment at least needs resolving?

Reply:
Whoop! Missed this one.